ACTION_ANIMATION_STEP jumps one frame too much

algmyr opened the pull request:
For all backends I tested (gdkpixbuf, libav, wand) pqiv jumps one frame too far when using ACTION_ANIMATION_STEP. Essentially for an argument `n=parameter.pint` animation_next_frame_fn is called `n` times in

```
	for(int i=0; i<parameter.pint; i++) {
		CURRENT_FILE->file_type->animation_next_frame_fn(CURRENT_FILE);
	}
```

and 1 time inside

```
	image_animation_timeout_callback(current_file_node);
```

This is easily solved by skipping `n-1` frames in the loop.

phillipberndt commented:
Thanks! Could you please add a comment hinting at the last frame progression taking place inside `image_animation_timeout_callback`? (Either before the `for` loop or before the `image_animation_timeout_callback` call, I don't care.) Also, feel free to add yourself as a contributor to the readme if you'd like.

algmyr commented:
Updated with comment and updated readme, hopefully the comment I added is clear enough. It's been a while since I had to rebase last, but it seems the squash worked fine so you'll have just have the single commit to merge.

While I'm at it, thanks for pqiv! It has been a very useful and hackable over the years! :)

phillipberndt commented:
Perfect. Thanks again!
